app/models/user.rb

```diff
@@ -0,0 +1,20 @@
+class User < ApplicationRecord
+  before_save :downcase_mail
+
+  validates :name,  presence: true,
+            length: {maximum: Settings[:user][:name_max_length]}
```

Comment:
anh suggest em dùng cú pháp kiểu này cho ngắn gọn nhé:
```suggestion
            length: {maximum: Settings.user.name_max_length}
```